### ISSUE TITLE
NIP-06 Support

### ISSUE THREAD

**Issue by rossbates** [jeffthibault/python-nostr#19]
Would it be possible to support key derivation from a seed phrase? Then from that support HD key derivation?

https://github.com/nostr-protocol/nips/blob/master/06.md

I realize this could potentially help spammers go nuts. The scenario I'm trying to accommodate though is seed and key management for things like truck fleets, sensors, and physical locations. One seed for many keys is the logical approach for that.

**Comment by jeffthibault** [comment]
Yeah, I can definitely add this. I was looking into it a while back and I think there are already BIP32 and BIP39 python packages that can be leveraged. I need to do a little more research first and then I'll add it.
